packages/aws-cdk-lib/custom-resource-config/test/custom-resource-config.test.ts

```diff
@@ -0,0 +1,547 @@
+import { Capture, Template, Match } from '../../assertions';
+import * as cdk from '../../core'
+import * as logs from '../../aws-logs';
+import * as s3 from '../../aws-s3';
+import * as ses from '../../aws-ses';
+import * as s3deploy from '../../aws-s3-deployment';
+import * as events from "../../aws-events";
+import * as targets from "../../aws-events-targets";
+import * as iam from "../../aws-iam";
+import { CustomResourceConfig } from '../custom-resource-config';
+
+describe('CustomResourceConfig empty retention', ()=> {
+    test('catch thrown error if no retention', ()=> {
+        const app = new cdk.App();
+        try {
+            CustomResourceConfig.of(app).addLogRetentionLifetime()
+        } catch (e) {
+            expect(e.message).toBe("Retention must have a value");
+        }
+    })
+})
+describe('aws-ses drop spams', () => {
+    test('Drop spams - UnModified by Aspects', () => {
```

Comment:
This test string gives no information about the feature. The use of Aspects is not relevant to the tests; test cases should describe behavior, not implementation details. 

In this case, the behavior is "When `addLogRetentionLifetime` is not called, no LogGroup is created". 

You don't need this case; every test we have for `ReceiptRuleSet` is already confirming that no LogGroup is created when you don't use `addLogRetentionLifetime`.